### ISSUE TITLE
Different data size

### ISSUE THREAD

**Issue by 311dada** [facebookresearch/ELI5#29]
**Bug description**
I follow the steps to construct the dataset. But my data size is 234171/9925/24820, different from 234063/9919/24804 mentioned in [issue #12](https://github.com/facebookresearch/ELI5/issues/12)

**Comment by JulesGM** [comment]
Eli5 is scraped from Reddit, I would assume that it would be expected that re-scraping the data would possibly not have the effect of reproducing the same data 1 to 1.
Use the KILT version instead for more reproducibility https://github.com/facebookresearch/ELI5 by using Huggingface https://huggingface.co/datasets/kilt_tasks
